docs/src/@primer/gatsby-theme-doctocat/live-code-scope.js

```diff
@@ -51,6 +52,7 @@ export default function resolveScope(metastring) {
     ...doctocatComponents,
     ...primerComponents,
     ...(metastring.includes('drafts') ? drafts : {}),
+    ...deprecated,
```

Comment:
This unintentionally reverses the change done in https://github.com/primer/react/blob/next/v35.0.0/docs/src/@primer/gatsby-theme-doctocat/live-code-scope.js#L55

Reply:
Fixed this.